data/json/mapgen_palettes/house_general_abandoned.json

```diff
@@ -0,0 +1,232 @@
+[
+    {
+      "type": "palette",
+      "id": "standard_domestic_palette_abandoned",
+      "//": "Intended as a palette for abandoned non-nested houses.  Symbols still open for use: 0 ! $ % & _ = ~ ? / , ` and some symbols that conflict with json (like brackets).",
+      "parameters": {
+        "interior_wall_type": {
+          "type": "ter_str_id",
+          "default": {
+            "distribution": [
+              [ "t_wall_b", 1 ],
+              [ "t_wall_g", 1 ],
+              [ "t_wall_p", 1 ],
+              [ "t_wall_P", 1 ],
+              [ "t_wall_r", 1 ],
+              [ "t_wall_w", 6 ],
+              [ "t_wall_y", 1 ]
+            ]
+          }
+        },
+        "exterior_wall_type": {
+          "type": "ter_str_id",
+          "default": {
+            "distribution": [
+              [ "t_brick_wall", 6 ],
+              [ "t_rock_wall", 3 ],
+              [ "t_wall_wood", 3 ],
+              [ "t_drystone_wall", 2 ],
+              [ "t_adobe_brick_wall", 4 ],
+              [ "t_concrete_wall", 4 ]
+            ]
+          }
+        },
+        "window_type": {
+          "type": "ter_str_id",
+          "default": {
+            "distribution": [
+              [ "t_window_domestic", 1 ],
+              [ "t_window_no_curtains", 1 ],
+              [ "t_window_no_curtains_open", 1 ],
+              [ "t_window_taped", 5 ],
+              [ "t_window_no_curtains_taped", 5 ],
+              [ "t_window_domestic_taped", 5 ],
+              [ "t_window_empty", 7 ],
+              [ "t_window_frame", 7 ],
+              [ "t_window_boarded", 7 ],
+              [ "t_window_boarded_noglass", 5 ],
+              [ "t_window_reinforced", 6 ],
+              [ "t_window_reinforced_noglass", 5 ],
+              [ "t_plastic_window", 7 ]
+            ]
+          }
+        },
+        "lawn_type": {
+          "type": "ter_str_id",
+          "default": {
+            "distribution": [
+              [ "t_region_groundcover_barren", 5 ],
+              [ "t_region_groundcover_forest", 3 ],
+              [ "t_region_groundcover", 1 ],
+              [ "t_region_groundcover_urban", 1 ]
+            ]
+          }
+        },
+        "house_fence_type": {
+          "type": "ter_str_id",
+          "default": {
+            "distribution": [
+              [ "t_splitrail_fence", 3 ],
+              [ "t_chainfence", 2 ],
+              [ "t_fence", 2 ],
+              [ "t_privacy_fence", 1 ],
+              [ "t_drystone_wall_half", 1 ]
+            ]
+          }
+        }
+      },
+      "toilets": { "t": {  } },
+      "furniture": {
+        "a": "f_fireplace",
+        "b": "f_bigmirror_b",
+        "c": "f_null",
+        "d": "f_null",
```

Comment:
Is there a reason to explicitly add null furniture?

Reply:
to ensure that furniture doesn't spawn when I call this palette.  Some I spawn and bash, some I chose not to spawn at all, mostly things people are less likely to leave behind.